dashboard/test/controllers/lessons_controller_test.rb

```diff
@@ -223,6 +226,20 @@ class LessonsControllerTest < ActionController::TestCase
     assert_equal 1, activity.position
   end
 
+  test 'update saves lesson name i18n in levelbuilder mode' do
```

Comment:
how is this checking that the lesson name is actually added to scripts.en.yml?

Reply:
good catch. it wasn't! it was only checking that the file was being written to. I've updated the test to verify that the lesson name is being written to both places.